holmes_operator/scheduler/manager.py

```
@@ -0,0 +1,305 @@
+            name: ScheduledHealthCheck name
+            namespace: ScheduledHealthCheck namespace
+        """
+        key = f"{namespace}/{name}"
+
+        if key in self.job_registry:
+            job_id = self.job_registry[key]
+            try:
+                self.scheduler.remove_job(job_id)
+                del self.job_registry[key]
+                logger.info(f"Removed schedule for {key}")
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)